bin/prettier.js

```diff
@@ -446,6 +452,26 @@ function writeOutput(result, options) {
 }
 
 function eachFilename(patterns, callback) {
+  // The ignorer will be used to filter file paths after the glob is checked,
+  // before any files are actually read
+  const ignoreFilePath = path.resolve(ignorePath);
+  let ignoreText = "";
+
+  try {
+    ignoreText = fs.readFileSync(ignoreFilePath, "utf8");
+  } catch (readError) {
+    if (readError.code !== "ENOENT") {
+      console.error(`Unable to read ${ignoreFilePath}:`, readError);
+      process.exit(2);
+    }
+  }
+
+  const ignorer = ignore();
+
+  if (ignoreText.trim()) {
```

Comment:
Seems to support both: https://github.com/kaelzhang/node-ignore#addignorefilepath